lib/generators/rolemodel/good_job/templates/config/initializers/good_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.good_job = {
+    smaller_number_is_higher_priority: true, # new default in V4 and for all of ActiveJob
+    queues: ENV.fetch('GOOD_JOB_QUEUES', 'pdf:1;-pdf'),
+    execution_mode: Rails.env.test? ? :inline : :external
```

Comment:
Aren't these defaults? Did you consider leaving them out?

Reply:
I think the default for development mode is `:async`.

Reply:
Ah, you're right. Which then raises the question: should the external process be added to the Procfile? Also curious why not use `:async` in development.

Reply:
> should the external process be added to the Procfile.dev?

I think this generator _is_ doing that already.

> Also curious why not use `:async` in development.

I would assume to make development more like production.

Reply:
Ah, I see that now

Reply:
We did consider leaving them out. We've had a couple of projects run into issues with `:async` in development. I think it is related to hot reloading when code changes are made causing things to hang. Instead of tracking that down, those projects have switched to `:external` for development. I've not experienced it myself but wanted to save people the troubles.